core/templates/pages/about-page/about-page.module.ts

```diff
@@ -54,10 +57,13 @@ import { ObjectsDomainConstants } from
     SharedComponentsModule
   ],
   declarations: [
-    ServiceBootstrapComponent
+    ServiceBootstrapComponent,
+    AboutPageComponent,
+    TranslatePipe
   ],
   entryComponents: [
-    ServiceBootstrapComponent
+    ServiceBootstrapComponent,
```

Comment:
Could you please check if we really need ServiceBootstrapComponent now?

Reply:
Yeah, with about page migrated this should not be required.

Reply:
Removed!